### DISCUSSION TITLE
Introduce Teamwork Between Organisms - Add a Communication Cell

### DISCUSSION THREAD

**Discussion by Floating-Adrift** [MaxRobinsonTheGreat/EvolutionSimulatorV2#122] [Ideas and Suggestions]
This cell could transmit from a list of signals which other communication cells within range can receive and act on.  The range would be a genetic variable.  Translation between species could be weighted towards only similar species understanding each other as well as whether they react positively or negatively.  Communications signals could be viewed as a heatmap, on a separate rendered layer, showing all of the messages being transmitted.  

Messages could just be directions, asking for assistance, etc. but could create some sort of teamwork with the right list of messages, and could even introduce mating between organisms.  The listening organism would need to determine how to react.

**Comment by Floating-Adrift** [comment]
I started writing another suggestion for a Pheremone cell, but it would essentially be the same as this one, functionally.

**Reply by gitergitgit** [reply]
releaser
